packages/demo/public/catalogues/catalogue-dktk-staging.json

```diff
@@ -1,5 +1,6 @@
 [
     {
+        "fieldType": "group",
```

Comment:
Similar to the change of the AST, do we have much improvement to add this breaking change? How would we migrate to it? Could we automate it, like with a script?